service/controller/v1/resource/drainer/create.go

```diff
@@ -33,19 +34,19 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 
 	if drainerConfig.Status.HasDrainedCondition() {
 		r.logger.LogCtx(ctx, "level", "debug", "message", "drainer config status has drained condition")
-		r.logger.LogCtx(ctx, "level", "debug", "message", "canceling resource for custom object")
+		r.logger.LogCtx(ctx, "level", "debug", "message", "canceling resource")
 
 		return nil
 	}
 
 	if drainerConfig.Status.HasTimeoutCondition() {
 		r.logger.LogCtx(ctx, "level", "debug", "message", "drainer config status has timeout condition")
-		r.logger.LogCtx(ctx, "level", "debug", "message", "canceling resource for custom object")
+		r.logger.LogCtx(ctx, "level", "debug", "message", "canceling resource")
 
 		return nil
 	}
 
-	if drainingTimedOut(drainerConfig, time.Now(), 30*time.Minute) {
+	if drainingTimedOut(drainerConfig, time.Now(), 10*time.Minute) {
```

Comment:
Bringing the timeout down to 10 minutes. Random. I think 30 is just too long? WDYT?

Reply:
Uncertain - what can go wrong if the timeout is too short?

Reply:
In theory some provider operator could create a drainer config without the `node-operator` being up for some reason. Once the `node-operator` gets its shit together again, it would not drain but set the timeout condition. This is already the case. It is just that the time of the operator being allowed to be down is not 30 minutes anymore, but now 10.

Reply:
@giantswarm/sre @giantswarm/sig-operator might have opinions here. I merge, but lets further talk if there is need.

Reply:
Is draining purely time-based now?

Reply:
Draining is event driven, based on the resync period. The timeout is time based.

Reply:
What we could do to try more often is to decrease the resync period to 3 minutes or something. It is 5 now AFAIK.